src/components/atoms/DatasetInfoBox.tsx

```diff
@@ -32,6 +39,17 @@ const DatasetInfoBox: React.FC<DatasetInfoBoxProps> = ({ title, description, tag
                     </Badge>
                 ))}
             </Pane>
+            <Pane marginTop="1rem" display="flex" flexDirection="column">
+                <Text size={300}>
+                    URL: <Link size={300}>{url}</Link>
+                </Text>
+                <Text size={300} marginTop="0.15rem">
+                    Last updated: <Strong size={300}>{formatDistanceToNow(updatedDate)}</Strong>
```

Comment:
Include suffix

https://date-fns.org/

It's on the front page

https://date-fns.org/

Reply:
Saying a data source was published "5 months" doesn't say much

Reply:
Oh oops, I will fix that. Could you send the link again?

Reply:
Yes of course sorry mister

https://date-fns.org/

Reply:
Thanks